UPGRADE-6.1.md

```diff
@@ -8,3 +8,5 @@ Serializer
  * Deprecate `ContextAwareDenormalizerInterface`, use `DenormalizerInterface` instead
  * Deprecate `ContextAwareEncoderInterface`, use `EncoderInterface` instead
  * Deprecate `ContextAwareDecoderInterface`, use `DecoderInterface` instead
+ * Deprecate supporting denormalization for `AbstractUid` in `UidNormalizer`, use one of `AbstractUid` child class instead
+ * Deprecate denormalizing to an abstract class in `UidNormalizer`
```

Comment:
to me this signals the same deprecation

Reply:
There is one deprecation for `supportsDenormalization()` that will return false instead of true for `AbstractUid::class` specifically.
And another for `denormalize()` that will throw an `\Error` if the class is abstract (instead of fallbacking to `Uuid::class`).